docs/core/common.md

```diff
@@ -121,3 +165,129 @@ public class DisposableBase : IDisposable
+    ```
+- メソッド引数はありとなしのどちらでも構いません。
+- 非同期処理には[UniTask](https://github.com/Cysharp/UniTask)を使ってください。
+- リトライするかしないかを例外で判断できるようにしてください（リトライ対象の処理失敗は例外をスローしてください）。
+
+リトライ処理の適用はRetryHandlerのOfメソッドを使います。
+OfメソッドによりRetryHandlerを生成してリトライ処理を適用したメソッドを呼び出せるようにします。
+Ofメソッドには次のパラメータを指定します。
+
+- リトライ対象メソッド
+- リトライするかしないかの判定処理
+- リトライ処理を制御する方法（リトライ戦略）
+- リトライをキャンセルするためのトークン
+
+リトライ処理を適用したメソッドの実行にはRetryHandlerのHandleAsyncメソッドを使います。
+HandleAsyncメソッドによりリトライ対象メソッドの処理が失敗した場合はリトライ処理が開始されます。
+HandleAsyncではリトライするかしないかの判定処理とリトライ戦略を使ってリトライの継続判定を行います。
+
+```csharp
+using var retryHandler = RetryHandler<Unit>.Of(RunAction, e => e is AccessViolationException, new CountingRetryStrategy());
+await sut.HandleAsync();
+```
+
+リトライ対象メソッドに引数がある場合は[ラムダ式](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/lambda-expressions)を使います。
+
+```csharp
+// Synchronous processing without return value
+using var retryHandler = RetryHandler<Unit>.Of(() => RunAction(value), e => e is AccessViolationException, new CountingRetryStrategy());
+await sut.HandleAsync();
+
+// Asynchronous processing without return value
+using var retryHandler = RetryHandler<Unit>.Of(async () => await RunActionAsync(value), e => e is AccessViolationException, new CountingRetryStrategy());
+await sut.HandleAsync();
+
+// Synchronous processing with return value
+using var retryHandler = RetryHandler<Unit>.Of(() => RunFunc(value), e => e is AccessViolationException, new CountingRetryStrategy());
+var result = await sut.HandleAsync();
+
+// Asynchronous processing with return value
+using var retryHandler = RetryHandler<Unit>.Of(async () => await RunFuncAsync(value), e => e is AccessViolationException, new CountingRetryStrategy());
+var result = await sut.HandleAsync();
+```
+
+#### リトライ戦略
+
+デフォルトで次のリトライ戦略を提供しています。
+
+- CountingRetryStrategy
+  - 回数でリトライを制御します。
+- NoRetryStrategy
+  - 常にリトライしません。
+  - リトライ処理を組み込んだライブラリを作る場合に使います。
+
+CountingRetryStrategyは最大リトライ回数とリトライ間隔を指定して使います。
+
+```csharp
+// Max retry count is 5, retry interval is 5 seconds each times
+var retryStrategy = new CountingRetryStrategy(5, _ => TimeSpan.FromSeconds(5));
+
+// Max retry count is 5, retry interval is retry count x 5 seconds
+var retryStrategy = new CountingRetryStrategy(5, retryCount => TimeSpan.FromSeconds(retryCount * 5));
+```
+
+CountingRetryStrategyはデフォルトでフィボナッチ数を使ったリトライ処理を提供します。
+デフォルトのフィボナッチ数は20個までなので最大リトライ回数を20より大きくしたい場合はリトライ間隔を指定してください。
+
+```csharp
+// Max retry count is 12(default), retry interval is Fibonacci numbers
+var retryStrategy = new CountingRetryStrategy();
+
+// Max retry count is 5, retry interval is Fibonacci numbers
+var retryStrategy = new CountingRetryStrategy(5);
+```
+
+リトライ戦略はIRetryStrategyインタフェースが提供します。
+デフォルトで提供しているクラスで対応できない場合はIRetryStrategyインタフェースを実装したクラスを作成してください。
+
+#### リトライ処理の再利用
+
+ボイスチャットやマルチプレイでは接続中に予期しない切断が発生する可能性があります。
+そのような切断が発生した場合は初回接続時の状態（チャンネル名などのアクセス情報）を使って再接続を行いたい場合があります。
+RetryHandlerはHandleAsyncメソッドが呼ばれる度にリトライ状態をリセットするため、再利用して使うことが可能です。
```

Comment:
Fixed!
https://github.com/extreal-dev/Extreal.Guide/pull/39/commits/f3c40f9159ce2b5ea68701156f78d1f9ef649fb0

Reply:
https://github.com/extreal-dev/Extreal.Guide/blob/c50c798f1268761f1608a23c0bf407576c362338/i18n/en/docusaurus-plugin-content-docs/current/core/common.md?plain=1#L275
併せてこちらも変更をお願いします。
it can be reused and used again. -> it can be reused.

Reply:
Fixed!
https://github.com/extreal-dev/Extreal.Guide/pull/39/commits/2d01c6fa8b47d35e49adb5c7c6311b073566c943

Reply:
確認しました！